source/directives/field-label.ts

```diff
@@ -86,6 +92,10 @@ module formFor {
     link($scope:FieldLabelScope, $element:ng.IAugmentedJQuery, $attributes:ng.IAttributes):void {
       $scope.attributes = $attributes;
 
+      $scope.$watch('useGlyphicons', function(useGlyphicons) {
```

Comment:
This doesn't look right. You're watching for changes to the `$scope.useGlyphicons` and then setting the same property to `FormForConfiguration.useGlyphicons`. Why?

Seems like you just want to set...

```js
$scope.useGlyphicons = formForConfiguration_.useGlyphicons;
```

And then not expose it as an externally-settable thing. (No need to add `useGlyphicons: '@'`.)